### PR TITLE
converted all .erb files to .haml

### PR DESCRIPTION
**Pull request by toocomplex1**
sent the code for review

### CODE REVIEW

**Comment by alienware** [comment]
When you work on removing the secrets.yml from git history, try the same thing with .DS_Store files.

**Comment by alienware** [comment]
Spacing is off in the new files, please convert tabs to 2 spaces. Also no not use extra newline spaces. Also, I hope you didn't use the gem that you added to do the conversions; otherwise good job getting onto it. But the spacings are unacceptable.

**Comment by alienware** [comment]
End of each file should have a new line.

File: blog/app/assets/stylesheets/bootstrap_and_overrides.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*
+  =require twitter-bootstrap-static/bootstrap
+
+  Use Font Awesome icons (default)
+  To use Glyphicons sprites instead of Font Awesome, replace with "require twitter-bootstrap-static/sprites"
+  =require twitter-bootstrap-static/fontawesome
+  */
```
Comment (alienware) [inline blog/app/assets/stylesheets/bootstrap_and_overrides.css:7]: Newline required.

File: blog/app/assets/stylesheets/articles.scss [hunk 2]
```diff
@@ -1,3 +1,10 @@
 // Place all the styles related to the articles controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.add-article-container {
+ 	background-color:white;
+ 	color:white;
+ 	padding:20px;
+ 	text-align:center;
+}
+
```
Comment (alienware) [inline blog/app/assets/stylesheets/articles.scss:10]: For CSS also, follow 2 spaces for tab.

File: blog/app/controllers/articles_controller.rb [hunk 3]
```diff
@@ -35,23 +35,16 @@ def create
     render 'new'
     end
   end
-
-
-  
-
```
Comment (alienware) [inline blog/app/controllers/articles_controller.rb:41]: Need one new line!

**Comment by alienware** [comment]
Fix spacing and newline isssue and it is ready to be merged.

File: blog/app/views/welcome/index.html.haml [hunk 4]
```diff
@@ -1,2 +1,5 @@
-%h1 Hello, Rails!
-= link_to 'My Blog', controller: 'articles'
+.container
+  .jumbotron
+    %h1 Welcome to toocomplex1 blog!
+.container.container-fluid
```
Comment (alienware) [inline blog/app/views/welcome/index.html.haml:4]: Here, use only .container

File: blog/app/views/comments/_comment.html.haml [hunk 5]
```diff
@@ -1,10 +1,15 @@
...
-    data: { confirm: 'Are you sure?' }                     |
+.panel.panel-default.arrow.left
+  .panel-body
+    %header.text-left
+      .comment-user
+        %i.fa.fa-user
+        = comment.commenter
+    .comment-post
+      %p
+        \:
```
Comment (alienware) [inline blog/app/views/comments/_comment.html.haml:9]: Aim at a little better aligned display for comments.

File: blog/app/views/articles/show.html.haml [hunk 6]
```diff
@@ -1,13 +1,12 @@
...
-%h2 Comments
-= render @article.comments
-%h2 Add a comment:
-= render 'comments/form'
-= link_to 'Edit', edit_article_path(@article)
-|
-= link_to 'Back', articles_path
+.container-fluid
+  .well
+    .blog-header
```
Comment (alienware) [inline blog/app/views/articles/show.html.haml:3]: Just for use sake, define this class and add some styling.

**Comment by toocomplex1** [comment]
done upto bootstrap

**Comment by alienware** [comment]
@toocomplex1 Use .container only once per page, if it is there in your application.html.haml, don't use it anywhere in your partials.

**Pull request merged**